app/src/main/java/org/fossasia/openevent/general/social/SocialLinksService.kt

```diff
@@ -0,0 +1,23 @@
+package org.fossasia.openevent.general.social
+
+import io.reactivex.Single
+
+class SocialLinksService(private val socialLinkApi: SocialLinkApi, private val socialLinksDao: SocialLinksDao) {
+
+    fun getSocialLinks(id: Long): Single<List<SocialLink>> {
```

Comment:
Should return flowable

Reply:
@iamareebjamal We used Single in all the previous cases, should we still use Flowable ?

Reply:
@iamareebjamal Should it remain as Single?

Reply:
We are always using Flowable AFAIK

Reply:
Actually in the discount codes fetching we use Single, even when dealing with users we are using Single actually.

Reply:
Okay I think I understood :+1: